### PR TITLE
Update k8s-multi-cluster.mdx

### LINKED ISSUE #3995: Changes in src/collections/resources/articles/kubernetes/multi-cluster/k8s-multi-cluster.mdx
### Current Behavior
If you take a look at the file you can see two import commands which are not exactly importing anything and if you edit the file you can also see a commented blog which should now be uncommented as the blog is now published.

image

### Desired Situation
Just remove the imported commands and uncomment the blog link so that it is visible.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ShivangShandilya**
**Description**

This PR fixes #3995 

**Notes for Reviewers**

The commented blog can now be viewed.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by thisiskaransgit — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 76b0fbe0de7ddf396e8d489f4177469efc965465 at: https://642c15fb22909c136842cf99--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 212928dd7115f882db66d81c95b7b36f826c1463 at: https://64301b70fd33ef40bdae09b5--layer5.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thank you, gents. I wonder if we might have this hyperlink changed to use the `<Link>` component. The `<Link>` component is to be used for all internal site links, while the `<a>` tags are for any external site links.

**Comment by ShivangShandilya** [comment]
> Thank you, gents. I wonder if we might have this hyperlink changed to use the `<Link>` component. The `<Link>` component is to be used for all internal site links, while the `<a>` tags are for any external site links.

hmm, I think we can do that

**Comment by ShivangShandilya** [comment]
> Thank you, gents. I wonder if we might have this hyperlink changed to use the `<Link>` component. The `<Link>` component is to be used for all internal site links, while the `<a>` tags are for any external site links.

@leecalcote I think I was able to do what you were asking for, mind taking a look if it's correct or not?

**Comment by l5io** [comment]
🚀 Preview for commit 78b62f6e9cf380f3a944e25df1a3ac3c8cb125e2 at: https://6432f4b55f3163426e5b856a--layer5.netlify.app

File: src/collections/resources/articles/kubernetes/multi-cluster/k8s-multi-cluster.mdx [hunk 1]
```diff
@@ -18,12 +18,7 @@ import { Link } from "gatsby";
 
 import { ResourcesWrapper } from "../../Resources.style.js";
 
-<!-- Unhide after the blog post below is published.  
-
-<div className="intro">
-  <p>Learn more about how to Link className="blog" to="/blog/meshery/multi-cluster-kubernetes-management-with meshery">wrangle multiple Kubernetes clusters with Meshery</Link.</p>
-</div>  -->
-
+Learn more about how to wrangle <Link to= "https://layer5.io/blog/meshery/multi-cluster-kubernetes-management-with-meshery">Multiple Kubernetes clusters with Meshery.</Link>
```
Comment (leecalcote) [inline src/collections/resources/articles/kubernetes/multi-cluster/k8s-multi-cluster.mdx:21]:
```suggestion
Learn more about how to wrangle <Link to= "/blog/meshery/multi-cluster-kubernetes-management-with-meshery">Multiple Kubernetes clusters with Meshery.</Link>
```

**Review by leecalcote — APPROVED** [review]
Given that they are all intra-site, the `https://layer5.io` portion is assumed, so we're safe to leave it out. 👍

**Comment by leecalcote** [comment]
Thank you, @ShivangShandilya 👍

**Comment by ShivangShandilya** [comment]
> Given that they are all intra-site, the `https://layer5.io` portion is assumed, so we're safe to leave it out. 👍

Noted 😁

**Pull request merged**